crowbar_framework/app/controllers/installer/upgrades_controller.rb

```diff
@@ -222,7 +222,7 @@ def nodes
           begin
             post_cleanup_repos
             @service_object.disable_non_core_proposals
-            @service_object.prepare_nodes_for_os_upgrade
+            @service_object.initiate_nodes_upgrade
```

Comment:
That would be the exception, catched later.

This is really the old code from 5-6 upgrade, no need to make it better. It's using the old controller action that won't be used in real 6-7 path.